src/generator.js

```diff
@@ -9,7 +9,7 @@ const composeUnit =
   (worldUnit, worldCurrencyName, code, roundingData) => {
     const unitName = composeUnitPrefix(worldCurrencyName) + ' ' + worldUnit['name'];
     return {
-      [code]: {
+      [(roundingData.shift === 2 ? 'cent' : code)]: {
```

Comment:
Maybe here we should check that code contains only valid ASCII symbols?
```
[(roundingData.shift === 2 ? 'cent' : validate_code(code))]
```
function `valiadte_code` should contain simple regexp-based check and just throw exception if string `code` is invalid